Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -40,8 +42,33 @@ namespace xr
     // Permission request ID used to uniquely identify our request in the callback when calling requestPermissions.
     const int PERMISSION_REQUEST_ID{ 8435 };
 
+    struct XrContextARCore : public IXrContextARCore {
+        bool Initialized{false};
+        ArSession* Session{nullptr};
```

Comment:
Where do we destroy the `ArSession` object? Since this object is now the holder/owner of it, would it make sense to move that ownership into the `XrContextARCore` destructor?

Reply:
its destroyed in the session impl destructor. I'll make sure its set to a nullptr, but i don't think we should change this life cycle behavior while folks are actively investigating xr life cycle bugs.